static/src/javascripts/projects/journalism/jtbd-survey.js

```diff
@@ -76,14 +70,14 @@ const save = (
     as: Answer[],
     q: Question,
     answer: Answer,
-    why: ?string
```

Comment:
`why?: string` is an optional param. What was the effect of `why: ?string`?

Reply:
the latter accepts `null` as a value. but in that case, it doesn't really matter. I don't know why I did this, will remove it 😋